server/README.md

```diff
@@ -0,0 +1,55 @@
+# Testing app backend 
+
+Express.js server provides access to report results. 
+
+---
+## Requirements
+- Node.js
+- Yarn
+
+
+## Install
+```bash 
+$ yarn install
+```
+
+## Configure app
+Application setup is done through environmental variables. 
+The following variables are used:
+
+### Connection with MongoDB
+* `MONGO_USERNAME`
+* `MONGO_PASSOWORD`
+* `MONGO_HOST`
+* `MONGO_PORT`
+* `MONGO_DATABASE`
+
+Are used to create mongodb connection url. 
+Collection used to store test harness reports is predetermined as `testexecutions`. 
+
+### Access restriction
+* `API_KEY_*`
+
+Env variables that regex matches `API_KEY_*` pattern are used to determine lists of API keys authorized to push report results to the storage. API keys have to be passed as an `x-api-key` header in incoming requests.
+
+### Other 
+* `PORT`
+
+By default, application runs on port `5000`.
+
+## Running the project
+
+    $ yarn dev
```

Comment:
```suggestion
$ yarn dev
```

Reply:
Indent is intentional for separate block for highlighting the block of code: 

    $ yarn dev

it has same effect as wrapping text with ``` block.